README.md

```diff
@@ -144,6 +144,11 @@ The `status` column enum values can be found in `org.pmiops.workbench.db.model.S
 
 ### UI
 
+In the UI, we use some [Fontawesome](https://fontawesome.com/) Pro icons. In order to install the Fontawesome Pro packages, you will need to download an `.npmrc` file with a Fontawesome license token. Copy it into the workbench root directory:
+```
+workbench$ gsutil cp gs://all-of-us-workbench-test-credentials/.npmrc .
```

Comment:
We want this to be part of the "first time repo setup", so I would also add this here (while the doc is under api, I don't think it's meant to be API specific): api/docs/developer-system-initialization.md